From ef5fa606472842e65ffd6105dcce8701a1fdc182 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Fri, 23 Aug 2024 16:01:39 +0300
Subject: [PATCH] [FSW-8694] fs1: add spi efuse manifest constants

This patch defines constant addresses required
for copying spi efuse manifest. SPI efuse is a
hybrid approach, it means spi efuse manifest is
located using a fake fuse boot descriptor from
either manifest 0 or 1.

Change-Id: Ie632599247714722df73f4265ca58b3b9a950014
---

diff --git a/arch/fs1/arc/soc.h b/arch/fs1/arc/soc.h
index 0926c58..a880052 100644
--- a/arch/fs1/arc/soc.h
+++ b/arch/fs1/arc/soc.h
@@ -128,4 +128,11 @@
 /* 32Bit arc viewpoint address of manifest 1 stored in ICCM */
 #define SPI_MANIFEST_IMG_1_DMA_DEST_ARC		(HL_FW_MEM_BASE + SPI_MANIFEST_IMG_1_OFFSET)
 
+/* For Fake fuse manifest */
+#define SPI_MANIFEST_IMG_2_ICCM_OFFSET	SZ_128K
+#define SPI_MANIFEST_IMG_2_OFFSET		(MNG_ARC_ICCM_OFFSET + SPI_MANIFEST_IMG_2_ICCM_OFFSET)
+#define SPI_MANIFEST_IMG_2_DMA_DEST		(FW_MEM_BASE_FULL + SPI_MANIFEST_IMG_2_OFFSET)
+/* 32Bit arc viewpoint address of manifest 2 stored in ICCM */
+#define SPI_MANIFEST_IMG_2_DMA_DEST_ARC		(HL_FW_MEM_BASE + SPI_MANIFEST_IMG_2_OFFSET)
+
 #endif /* _SOC_H_ */
